duckduckgo/BrowserServicesKit
# Advance content-scope-scripts forward

Task/Issue URL: https://app.asana.com/0/569473074191537/1201894298881763/f
Tech Design URL:
CC:

**Description**:

Advances content scope scripts forward which isolates globals from the page when we don't intend to modify them.
- This repo now has a test to prevent this happening again.

**Steps to test this PR**:
1. Open https://good.third-party.site/security/js-leaks.html
2. Click Safari
3. Click check

Check for globals that look defined by us (the test gets outdated per Safari release: but previously we had 'init' and 'contentScopeFeatures' leaked)



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
